kore/test/Test/Kore/Unparser.hs

```diff
@@ -216,21 +218,21 @@ test_parse =
         ]
 
 roundtrip ::
-    (HasCallStack, Unparse a, Eq a, Show a) => Gen a -> Parser a -> Property
+    (HasCallStack, Unparse a, Eq a, Show a) => Gen a -> (FilePath -> Text -> Either String a) -> Property
```

Comment:
Can we have something like `type Parser = (FilePath -> Text -> Either String a)` in order to avoid this long type signature?